### PR TITLE
Simplify unvetted record URL.

### LINKED ISSUE #2344: Simplify unvetted record URL
The current URL for an unvetted record is `/record/unvetted/91cfcc8c62af2022`.  The `unvetted/` part of this was leftover from when the backend required you to specify whether a record was unvetted or vetted in order to retreive it.  This was removed from the backend design, but was not removed from the URL.  Since you only need to provide the backend with the token to retrieve the record, the URL should simply be `/record/[token]`.

### LINKED ISSUE #2350: Edit: Paywall error is displayed while having paywall disabled.
image

To reproduce:
1. Create a proposal while having paywall disabled. 
2. Click on edit.
3. Refresh page.
4. Paywall message will show up and it shouldn't.

### LINKED ISSUE #2351: Edit: Updated propsoal not persisted.
When editing a proposal for the second time the proposal form displays old form values.

Related to #2365.

### PR DESCRIPTION
**Pull request by amassarwi**
Closes #2344.

### CODE REVIEW

File: src/containers/Proposal/Detail/Detail.jsx [hunk 1]
```diff
@@ -62,15 +62,16 @@ const ProposalDetail = ({ Main, match, state }) => {
             )}
             {!isCensoredProposal(proposal) && (
               <Comments
-                recordAuthorID={proposal?.userid} // this will be deprecated after tlog
+                // this will be deprecated after tlog
```
Comment (tiagoalvesdulce) [inline src/containers/Proposal/Detail/Detail.jsx:65]: tlog is in, can we deprecate it?

File: src/pages/Routes.jsx [hunk 2]
```diff
@@ -43,52 +40,34 @@ const Routes = ({ location }) => {
           />
           {/* Record routes */}
           <AdminAuthenticatedRoute
-            path={"/record/admin"}
+            path="/record/admin"
```
Comment (tiagoalvesdulce) [inline src/pages/Routes.jsx:43]: wouldn't be better if we use `admin/records` instead?

**Review by tiagoalvesdulce — COMMENTED** [review]
Code is looking good, left 2 comments

File: src/containers/Proposal/Detail/Detail.jsx (hunk 1 above)
Comment (amassarwi) [reply src/containers/Proposal/Detail/Detail.jsx:65]: good quesstion, let me check what does it mean hehe might be outdated comment ? 🤔

**Comment by amassarwi** [comment]
@tiagoalvesdulce done mate and on the way found new issues with the admin proposals page:
https://github.com/decred/politeiagui/issues/2361

**Review by tiagoalvesdulce — APPROVED** [review]

**Pull request merged**